core/core_modules/help.js

```diff
@@ -63,15 +64,14 @@ exports.run = function(api, event) {
         context = {
             commandPrefix: api.commandPrefix
         },
-        help;
-
-    if (commands.length === 1) {
-        help = shortSummary.call(exports.platform, context);
-    }
-    else {
-        commands.splice(0, 1);
-        help = longDescription.call(exports.platform, commands.join(' '), context);
-    }
+		help;
+	if (commands.length === 1) {
+		help = shortSummary.call(exports.platform, context, event);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2651361708/issues/source?bid=3282987&fileBranchId=3324570#l69)